src/Game.hs

```diff
@@ -97,17 +96,20 @@ assignCardPl numCards currentState@(remCards,p:ps)
 initialDeck :: [Card]
 initialDeck = [Card st val | st <- [Spades .. Diamonds],val <- [Two .. Ace]]
 nipun :: Player
-nipun = Player "Nipun" []
+nipun = Player "Nipun" "1" []
 anish :: Player
-anish = Player "Anish" []
+anish = Player "Anish" "2" []
 mahesh :: Player
-mahesh = Player "Mahesh" []
+mahesh = Player "Mahesh" "3" []
 suresh :: Player
-suresh = Player "Suresh" []
+suresh = Player "Suresh" "4" []
 
 allPlayers :: [Player]
 allPlayers = [nipun,anish,mahesh,suresh]
 
+-- PlayerState to send to the client of player for displaying on UI
+getPlayerState :: Gamestate -> Player -> PlayerState
+getPlayerState gs player = PlayerState (name player) (_id player) (cards player) (currentRound gs) (round gs) (teams gs) (trump gs)
```

Comment:
```suggestion
getPlayerState gs player = PlayerState player (currentRound gs) (round gs) (teams gs) (trump gs)
```

Reply:
yeah we can do that, missed it, would commit the suggestion